docs/overview/deep_dive/data_types.rst

```diff
@@ -474,6 +474,68 @@ However, torch does not support ``uint32``, and so we cannot fully adhere to the
+then we simply identify that by simply replacing the different float dtypes with the str `float`. The same logic is applied to other dtypes such as `complex`, where we simply replace the entire dtypes with the str `complex`
+
+An example is :func:`ivy.fmin` with a tensorflow backend:
+
+.. code-block:: python
+
+    @with_supported_dtypes({"2.13.0 and below": ("float",)}, backend_version)
+    def fmin(
+        x1: Union[tf.Tensor, tf.Variable],
+        x2: Union[tf.Tensor, tf.Variable],
+        /,
+        *,
+        out: Optional[Union[tf.Tensor, tf.Variable]] = None,
+    ) -> Union[tf.Tensor, tf.Variable]:
+        x1, x2 = promote_types_of_inputs(x1, x2)
+        x1 = tf.where(tf.math.is_nan(x1), x2, x1)
+        x2 = tf.where(tf.math.is_nan(x2), x1, x2)
+        ret = tf.experimental.numpy.minimum(x1, x2)
+        return ret
+
+As seen in the above code, we simply use the str `float` instead of writing all the float dtypes that are supported
+
+Another example is :func:`ivy.floor_divide` with a tensorflow backend:
+
+.. code-block:: python
+
+    @with_unsupported_dtypes({"2.13.0 and below": ("complex",)}, backend_version)
+    def floor_divide(
+        x1: Union[float, tf.Tensor, tf.Variable],
+        x2: Union[float, tf.Tensor, tf.Variable],
+        /,
+        *,
+        out: Optional[Union[tf.Tensor, tf.Variable]] = None,
+    ) -> Union[tf.Tensor, tf.Variable]:
+        x1, x2 = ivy.promote_types_of_inputs(x1, x2)
+        return tf.experimental.numpy.floor_divide(x1, x2)
+
+As seen in the above code, we simply use the str `complex` instead of writing all the complex dtypes that are not supported
+
+Supported and Unsupported Data Types Attributes
+------------------------------------
+
+In addition to the unsupported / supported data types decorator, we also have the :attr:`unsupported_dtypes` and :attr:`supported_dtypes` attributes. These attributes operate in a manner similar to the attr:`@with_unsupported_dtypes` and attr:`@with_supported_dtypes` decorators.
+However, the major difference between the attributes and the decorators is that the attributes are assigned and used within the ivy functional API :mod:`ivy/functional/ivy/<ivy_functional_API>` for functions that don't have a specific backend implementation for each backend,
+while the decorators are used within the frontend :mod:`ivy/functional/frontends/<some_frontend>` and backend :mod:`ivy/functional/backends/<some_backend>` to identify the supported or unsupported data types, depending on the use case.
+
+An example of a function which does not have a specific backend implementation for each backend is the :attr:`einops_reduce` function. `This function <https://github.com/unifyai/ivy/blob/8516d3f12a8dfc4ec5f819789937d196c7e28566/ivy/functional/ivy/general.py#L1964>`_ , makes use of a third-party library :attr:`einops` which has these backend-specific implementations.
```

Comment:
it would be good to have this under a sub-header like `Special Case` or something so that someone seeing this explanation knows that they'd most likely not need to use this way of specifying unsupported dtypes. Also it would be useful to highlight that we're setting the attribute in the `ivy` function itself.
Also, I think we can change ` which has these backend-specific implementations.` to ` which has its own backend-agnostic implementations`